app/views/orders/set.html.erb

```diff
@@ -2,18 +2,62 @@
 
 <%= bootstrap_form_for work_order, url: wizard_path, method: :put  do |f| %>
 
+<table class="table table-striped table-condensed" id="set-list-table">
+  <tr>
+    <thead>
+      <th></th>
+      <th>Submission Name</th>
+      <th>Size</th>
+      <th>Created</th>
+      <th>ID</th>
+    </thead>
+  </tr>
+
```

Comment:
I suggest removing the `<tr>` element and place` <thead>` directly under `<table`>, to follow the specification:
https://www.w3.org/TR/html5/tabular-data.html#the-thead-element